dotCMS/src/main/java/com/liferay/portal/model/User.java

```diff
@@ -415,4 +419,13 @@ public boolean hasConsoleAccess() {
 	private User _user;
 	private Date modificationDate;
 
+	@JsonIgnore
+	@Override
+	public ManifestInfo getManifestInfo(){
+		return new ManifestInfo(map(
+			"object type", PusheableAsset.USER.getType(),
+			"id", this.getUserId(),
+			"title", this.getFullName()
+		));

```

Comment:
I think just fields (no internal map) would be better... `new ManifestInfo(PusheableAsset.USER.getType(), this.getUserId(), this.getFullName())`

Reply:
code removed